src/Template/Element/Form/relations.twig

```diff
@@ -25,22 +24,30 @@
                         <span class="status">status</span>
                     </div>
 
-                    {# Loader for Load button #}
-                    {% element 'loader' %}
-
                     {# related objects #}
                     <div class="related-objects-list">
-                        <div class="related-object" v-for="related in objects" v-show="!containsId(removedRelated, related.id)">
-                            {# <span class="priority"><% related.meta.relation.priority %></span> #}
-                            <span class="title"><% related.attributes.title || related.attributes.name %></span>
-                            <span class="id"><% related.id %></span>
-                            <span class="uname"><% related.attributes.uname  %></span>
-                            <span class="status"><% related.attributes.status %></span>
-                            <a class="open" v-bind:href="buildViewUrl(related.type, related.id)" target="_blank">Open</a>
-                            <button class="remove" @click.prevent="removeRelations(related.id, related.type)">x</button>
-                        </div>
+                        <staggered-list stagger="25">
+                            <div class="related-object"
```

Comment:
Maybe we should handle the case of `related.attributes` empty (adding an `{% if %}`-`{% else %}`-`{% endif %}`).